also include alphafold.data.tools and alphafold.model.tf, which are not picked up by find_packages()
author: Kenneth Hoste (HPC-UGent)
--- alphafold-2.0.0/setup.py.orig	2021-08-14 08:42:43.030919330 +0200
+++ alphafold-2.0.0/setup.py	2021-08-14 08:43:59.183245602 +0200
@@ -26,7 +26,7 @@
     author_email='alphafold@deepmind.com',
     license='Apache License, Version 2.0',
     url='https://github.com/deepmind/alphafold',
-    packages=find_packages(),
+    packages=find_packages() + ['alphafold/data/tools', 'alphafold/model/tf'],
     install_requires=[
         'absl-py',
         'biopython',
